hcl2template/types.packer_config.go

```diff
@@ -7,13 +7,13 @@ import (
 	"github.com/hashicorp/packer/helper/common"
 	"github.com/hashicorp/packer/packer"
 	"github.com/zclconf/go-cty/cty"
-
-	"github.com/gobwas/glob"
 )
 
 // PackerConfig represents a loaded Packer HCL config. It will contain
 // references to all possible blocks of the allowed configuration.
 type PackerConfig struct {
+	// parser *Parser
```

Comment:
is this forgotten here?

Reply:
ah, we will need it in the future for the `fix` command but the linting didn't like it lol so here we are.